docs/pages/concept/web.md

```diff
@@ -0,0 +1,148 @@
+# Web Renderer
+## Overview
```

Comment:
[NIT]
```suggestion
# Web Renderer

## Overview
```

Reply:
If you write this in VS Code, check out markdownlint https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint